basics/getting-started/frequent-questions/operations-faq.md

```diff
@@ -92,6 +92,39 @@ curl -X POST "http://localhost:9000/tenants"
 -H "Content-Type: application/json" 
 -d "{\"tenantRole\":\"BROKER\",\"tenantName\":\"foo\",\"numberOfInstances\":1}"
 ```
+### How to I manually run a Periodic Task
+
+Use the `GET /periodictask/names` API to fetch the names of all the Periodic Tasks running on your Pinot cluster.
+
+```
+curl -X GET "http://localhost:9000/periodictask/names" -H "accept: application/json"
+
+[
+  "RetentionManager",
+  "OfflineSegmentIntervalChecker",
+  "RealtimeSegmentValidationManager",
+  "BrokerResourceValidationManager",
+  "SegmentStatusChecker",
+  "SegmentRelocator",
+  "MinionInstancesCleanupTask",
+  "TaskMetricsEmitter"
+]
+```
+
+To manually run a named Periodic Task use the `GET /periodictask/run` API
+
+```
+curl -X GET "http://localhost:9000/periodictask/run?taskname=SegmentStatusChecker&tableName=jsontypetable&type=OFFLINE" -H "accept: application/json"
+
+{
+  "Log Request Id": "api-09630c07",
```

Comment:
Where does `api-09630c07` come from? Is this something we control, if so, need better naming there.